apps/web/src/app/(networks)/(evm)/[chainId]/(positions)/pool/page.tsx

```diff
@@ -1,25 +1,37 @@
-'use client'
-
 import { Container } from '@sushiswap/ui'
 import { notFound } from 'next/navigation'
-import React, { use } from 'react'
+import React from 'react'
 import { TableFiltersNetwork } from 'src/app/(networks)/_ui/table-filters-network'
 import { TableFiltersSearchToken } from 'src/app/(networks)/_ui/table-filters-search-token'
 import { POOL_SUPPORTED_NETWORKS } from 'src/config'
-import { type EvmChainId, isSushiSwapChainId } from 'sushi/evm'
+import { isPublicBladeChainId } from 'src/config.server'
+import {
+  SushiSwapProtocol,
+  isBladeChainId,
+  isSushiSwapChainId,
+} from 'sushi/evm'
 import { TableFiltersResetButton } from '~evm/[chainId]/_ui/table-filters-reset-button'
 import { PositionsTab } from './_ui/positions-tab'
 
-export default function MyPositionsPage(props: {
+export default async function MyPositionsPage(props: {
   params: Promise<{ chainId: string }>
 }) {
-  const params = use(props.params)
-  const chainId = +params.chainId as EvmChainId
+  const params = await props.params
+  const chainId = +params.chainId
+
+  const isBladeChain =
+    isBladeChainId(chainId) && (await isPublicBladeChainId(chainId))
```

Comment:
you can probably remove all `isBladeChainId` checks if you're also calling `isPublicBladeChainId`, since the former is included in the latter